exe/bookroom.spec

```diff
@@ -0,0 +1,44 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+block_cipher = None
+
+
+a = Analysis(['/Users/ccho/Developer/libraryroomscheduler/proofOC/bookRoom.py'],
```

Comment:
Having these as absolute file paths causes the build to fail, since my local copy of the project doesn't live in `/Users/ccho/Developer`